src/Handler/TagHandler.php

```diff
@@ -96,6 +100,9 @@ static function ($tag) use ($repoPrefix) {
                 $tagsString = trim(substr($tagsString, 0, strrpos(
                     substr($tagsString, 0, $this->tagsHeaderMaxLength + 1), ' '
                 )));
+                if ($this->tagsHeaderReducedTTl && !$response->headers->getCacheControlDirective('s-maxage')) {
```

Comment:
Shoulnt this be:
```suggestion
                if ($this->tagsHeaderReducedTTl && $response->headers->getCacheControlDirective('s-maxage')) {
```

As in, if there is already a TTL set by the prev event, then we know it's cacheable and we need to reduce the TTL
Could also do `$response->headers->getCacheControlDirective('s-maxage') > $this->tagsHeaderReducedTTl` to be sure it's larger

Reply:
Ah, you are right, good catch. Thanks!